src/extension.ts

```diff
@@ -83,6 +87,11 @@ export function activate(context: ExtensionContext) {
 
             if (HIDE_WINDOW_OUTPUT) {
```

Comment:
We won't have access to `childProcess.stderr` there and generally we won't be able to access `LanguageClient._childProcess`, because it's private and also [not assigned](https://github.com/Microsoft/vscode-languageserver-node/blob/master/client/src/main.ts#L178) (case where we spawn server with a closure)